src/pages/products/sgid/society/state-fuel-sites.astro

```diff
@@ -26,11 +26,22 @@ export const metadata: IMetadata = {
 
 const page: IPageMetadata = {
   ...metadata,
-  updateHistory: [`December 2023`],
-  hub: {
-    ...dataPages[metadata.pageTitle],
-  },
-  config: { skipInDepth: true },
+  application: 'https://experience.arcgis.com/experience/9a38f59e6f914367b18fbbde037028e2',
+  updateHistory: [`AS NEEDED`],
+  links: [
+  {
+      title: 'Utah State Fuel Sites in the SGID on ArcGIS',
+      url: 'https://opendata.gis.utah.gov/datasets/utah-state-fuel-sites/about',
+    },
+    {
+      title: 'File Geodatabase',
+      url: 'https://opendata.arcgis.com/datasets/8d9bc67285fb421fb15d55342148ac1e_0.gdb',
+    },
+    {
+      title: 'Shapefile',
+      url: 'https://opendata.arcgis.com/datasets/8d9bc67285fb421fb15d55342148ac1e_0.zip',
+    },
+  ],
```

Comment:
i don't understand why this needs to change if it's continuing as an open dataset? What about the internal data influences this change?

Reply:
if it is removed from Internal the Open SGID sample query will be invalid

Reply:
If the [openSgid](https://github.com/agrc/gis.utah.gov/blob/main/src/components/data/HubDownloads.astro#L46) prop from the downloadMetadata is null it won't show that.